scipy/stats/tests/test_multivariate.py

```diff
@@ -27,6 +28,7 @@
 from scipy.stats import norm, uniform
 from scipy.stats import ks_2samp, kstest
 from scipy.stats import binom
+from scipy.stats import hypergeom
```

Comment:
can we put these under one import statement?

Reply:
Sure. Will commit this soon.

Reply:
fixed.